src/mds/query.py

```diff
@@ -68,13 +74,30 @@ async def search_metadata(
     """
     limit = min(limit, 2000)
     queries = {}
+    include_internal_id = internal_id
     for key, value in request.query_params.multi_items():
-        if key not in {"data", "limit", "offset"}:
+        if key not in {"data", "limit", "offset", "internal_id"}:
```

Comment:
hitting `/metadata?gen3_internal_id` returns a 500 (i'm not sure what result i expected, but not a 500 😛)

hitting `/metadata?internal_id` returns:
```
{
  detail: [
    {
      loc: [
        "query",
        "internal_id",
      ],
      msg: "value could not be parsed to a boolean",
      type: "type_error.bool",
    }
  ]
}
```